packages/modules/itwin-viewer-react/package.json

```diff
@@ -26,9 +26,9 @@
     "copy-images": "concurrently npm:dev-copy-images npm:prod-copy-images",
     "dev-copy-images": "copyfiles -u1 \"./src/images/**\" ./build",
     "prod-copy-images": "copyfiles -u1 \"./src/images/**\" ./lib",
-    "copy-json": "concurrently npm:dev-copy-json npm:prod-copy-json ",
-    "dev-copy-json": "copyfiles -u1 \"./src/**/*.json\" ./build",
-    "prod-copy-json": "copyfiles -u1 \"./src/**/*.json\" ./lib",
+    "copy-public": "concurrently npm:dev-copy-public npm:prod-copy-public",
```

Comment:
This is probably going to cause an issue with the tree we have in the viewer. 
There is the ruleset file which we need to make sure gets copied over: packages\modules\itwin-viewer-react\src\components\app-ui\tree\Tree.ruleset.json

One solution could be using our tree-widget-react pkg instead of having our own implementation

Reply:
yep....you're right. I swear I searched for .json.....I'll update

Reply:
I didn't even know we had a tree pkg. I'll add a PBI for that one as well

Reply:
fixed

Reply:
Guess it still worked locally due to monorepo symlinking magic, but sounds good.